insights/parsers/package_provides_httpd.py

```diff
@@ -0,0 +1,63 @@
+"""
+PackageProvidesHttpd - command ``/bin/echo {httpd_command_package}``
+====================================================================
+
+This command reads the output of the pre-command:
+
+``for jp in `/bin/ps auxwww | grep httpd | grep -v grep| awk '{print $11}' | sort -u`; do echo $jp | xargs rpm -qf`; done``
```

Comment:
Would it make sense to just make sure the following paragraph just describes what the datasource does since a developer will not find this actual command anywhere in the specs?